### PR TITLE
Adding articles PySpark and Decimator

### PR DESCRIPTION
**Pull request by FlorianJacta**
- Add PySpark article
- Add Decimator article
- Fix error 404 for Long Running Callbacks
- Change name of the Tutorial (Build a Complete app to First Realistic Application)
- Small changes on Databricks tip

Pre-built articles for PySpark and Decimator. Download and open it, you should be able to see the built version of the articles:
[Articles Pre-built.zip](https://github.com/Avaiga/taipy-doc/files/13991384/Articles.Pre-built.zip)

image

image

image

image

### CODE REVIEW

File: docs/knowledge_base/tips/databricks/index.md [hunk 1]
```diff
@@ -40,8 +39,8 @@ have widgets/parameters and get the results of your job.
 Databricks job. This code can include data processing, analysis, or any other tasks you 
 need to perform.
 
-Here is an example of a Notebook where parameters are passed to the job and results are 
-then being retrieved:
+Here is an example of a Databrick Notebook where parameters are passed to the job
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/databricks/index.md]:
```suggestion
Here is an example of a DatabrickS Notebook where parameters are passed to the job 
```

**Comment by marisogo** [comment]
Update image of databricks tips
dbricks_img

File: docs/knowledge_base/tips/databricks/index.md [hunk 2]
```diff
@@ -62,7 +70,7 @@ dbutils.notebook.exit(data[:5])
 ```
 
 `dbutils.widgets.get("param1")`: is how you can get the parameters passed to your job. 
-Note that results and parameters are stringified. No complex objects can be passed 
+Note that results and parameters are stringified. All JSON-compatible objects can be passed
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/databricks/index.md]:
```suggestion
Note that results and parameters are stringified. Only JSON-serializable objects can be passed 
```

File: docs/knowledge_base/tips/databricks/index.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
 This tip shows how to integrate Taipy scenarios with platforms like Databricks. 
 Taipy scenarios serve as a potent tool for orchestrating tasks and examining various 
 versions of a business problem. In this article, we'll look into the integration of 
 Databricks jobs with Taipy scenarios, showcasing how this can elevate your data 
 processing capabilities.
 
-TO DO, put illustration
 ![Databricks](databricks.png){width=100%}
 
 ## Scenarios and Databricks Integration
```
Comment (jrobinAV) [inline docs/knowledge_base/tips/databricks/index.md]: This should be a H1

File: docs/knowledge_base/tips/databricks/index.md
Comment (jrobinAV) [inline docs/knowledge_base/tips/databricks/index.md:1]: I would not make the steps as a title. Just a numerated list.

File: docs/knowledge_base/tips/decimator/index.md
Comment (jrobinAV) [inline docs/knowledge_base/tips/decimator/index.md:1]: image
The table of content looks messy and not really readable

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
At least a typo to fix

File: docs/knowledge_base/tips/big_data_models/index.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ enables effortless management of large datasets
 This article focuses on the seamless integration of Dask (for handling out-of-core data)
  with Taipy, a Python library used for pipeline orchestration and scenario management.
 
-# 1. Sample Application
+# Sample Application
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/big_data_models/index.md:19]: Please skip a line after the title, like you did everywhere else. Homogeneity and clarity

**Review by FabienLelaquais — APPROVED** [review]
👍

**Comment by jrobinAV** [comment]
Please delete the branch when it is not used anymore.

**Pull request merged**